src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -25,21 +26,21 @@
 // */
 //public class DeleteCommandTest {
 //
-//    private Model model = new ModelManager(getTypicalInventory(), new UserPrefs());
+//    private Model model = new ModelManager(getTypicalInventory(), new UserPrefs(), new RecipeBook());
 //
-////    @Test
-////    public void execute_validIndexUnfilteredList_success() {
+//    @Test
+//    public void execute_validIndexUnfilteredList_success() {
 //        Ingredient ingredientToDelete = model.getFilteredIngredientList().get(INDEX_FIRST_INGREDIENT.getZeroBased());
-////        DeleteCommand deleteCommand = new DeleteCommand(INDEX_FIRST_INGREDIENT);
-////
-////        String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_INGREDIENT_SUCCESS,
-////                Messages.format(ingredientToDelete));
-////
-////        ModelManager expectedModel = new ModelManager(model.getInventory(), new UserPrefs());
-////        expectedModel.deleteIngredient(ingredientToDelete);
-////
-////        assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
-////    }
+//        DeleteCommand deleteCommand = new DeleteCommand(INDEX_FIRST_INGREDIENT);
+//
+//        String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_INGREDIENT_SUCCESS,
+//                Messages.format(ingredientToDelete));
+//
+//        ModelManager expectedModel = new ModelManager(model.getInventory(), new UserPrefs());
+//        expectedModel.deleteIngredient(ingredientToDelete);
```

Comment:
Not sure if this actually checks that the item is missing because how the expected model is generated uses the same function of deleteIngredient. deleteIngredient function from the model could be faulty.